### PR TITLE
Add notes on Firestore indexes

### PR DESCRIPTION
**Pull request by nakabonne**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: docs/content/en/docs/operator-manual/control-plane/installation.md [hunk 1]
```diff
@@ -62,6 +62,8 @@ spec:
       credentialsFile: /etc/pipecd-secret/gcs-service-account
 ```
 
+You need to prepare the service account with `Cloud Datastore User` and also `Cloud Datastore Index Admin` roles. It's because PipeCD automatically creates needed Firestore composite indexes.
```
Comment (khanhtc1202) [inline docs/content/en/docs/operator-manual/control-plane/installation.md:65]: nit
```
PipeCD automatically creates needed Firestore composite indexes in the background. You have to prepare the service account with `Cloud Datastore User` and `Cloud Datastore Index Admin` roles to perform this behavior.
```

**Comment by nakabonne** [comment]
Modified taking your advice!

**Comment by khanhtc1202** [comment]
> Modified taking your advice!

Thanks 🙏

**Comment by khanhtc1202** [comment]
😁 
/approve

**Pull request merged**